lib/src/utils/helpers.dart

```diff
@@ -62,6 +62,10 @@ String assignVersionWeight(String version) {
     version = version.replaceFirst('v', '');
   }
 
+  if (version.contains('custom_')) {
```

Comment:
Yes, it only handle when after custom is correct version syntax. I think just remove `custom_` like `v` so in case custom_1.17.4 then it should work like 1.17.4 use hasOldBinPath. if like `isGitCommit` then it always use new binPath?